### PR TITLE
Adds a piece of text to the Readme.md

### PR DESCRIPTION
**Pull request by tijspdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Mixer
  mixer app random words
+
+ testtesttest
```
Comment (bartp1990) [inline README.md]: Are you sure this is spelled correctly?

Comment (tijspdev) [reply README.md]: Thanks will do!

Comment (bartp1990) [reply README.md]: [Fixed in commit 
cac73f5](https://github.com/tijspdev/Mixer/pull/1/commits/cac73f51c677684a92af9faa6b7d5a156fcaf619)

Comment (bartp1990) [reply README.md]: Ok looks amazing!

**Pull request merged**